compat/test/browser/forwardRef.test.js

```diff
@@ -392,7 +392,7 @@ describe('forwardRef', () => {
 		expect(_ref.current).to.equal(scratch.firstChild);
 	});
 
-	it('should forward at diff time instead vnode-creation.', () => {
+	it.skip('should forward at diff time instead vnode-creation.', () => {
```

Comment:
Same here

Reply:
I guess I should just remove these skipped tests, will have a deeper look whether they could still matter